tendrl/monitoring_integration/alert/handlers/cluster/gluster/brick_utilization_handler.py

```diff
@@ -0,0 +1,139 @@
+from etcd import EtcdKeyNotFound
+from subprocess import CalledProcessError
+from tendrl.commons.event import Event
+from tendrl.commons.message import ExceptionMessage
+from tendrl.monitoring_integration.alert import constants
+from tendrl.monitoring_integration.alert.handlers import AlertHandler
+from tendrl.monitoring_integration.alert import utils
+from tendrl.monitoring_integration.alert.exceptions import InvalidAlertSeverity
+from tendrl.monitoring_integration.alert.exceptions import NodeNotFound
+
+
+class BrickHandler(AlertHandler):
+
+    handles = 'brick'
+    representive_name = 'bricks_utilization_alert'
+
+    def __init__(self):
+        AlertHandler.__init__(self)
+
+    def format_alert(self, alert_json):
+        alert  = self.parse_alert_metrics(alert_json)
+        try:
+            alert["alert_id"] = None
+            alert["node_id"] = utils.find_node_id(
+                alert['tags']['integration_id'],
+                alert['tags']['fqdn']
+            )
+            alert["time_stamp"] = alert_json['NewStateDate']
+            alert["resource"] = self.representive_name
+            alert['alert_type'] = constants.ALERT_TYPE 
+            alert['severity'] = constants.TENDRL_GRAFANA_SEVERITY_MAP[
+                alert_json['State']]
+            alert['significance'] = constants.SIGNIFICANCE_HIGH
```

Comment:
Should it always be HIGH?

Reply:
yes, @anmolbabu this is always hight right?